owncloud/web
# ODS design tokens are not aligned with Figma Design Colors

## Description
Colors need to be double-checked and adjusted, especially in dark mode.

![181737446-9ebe73f0-b912-4ca9-8585-dcedb08385f9](https://user-images.githubusercontent.com/42426067/182813477-d68de63e-7351-4d83-9329-075f46a40555.png)

## Related Issue
- [Fixes #7373 ](https://github.com/owncloud/web/issues/7373)
## Motivation and Context


## How Has This Been Tested?



- test environment: Firefox Dev Tools
- inspect site with Firefox Dev Tools > Tab "Accessibility" > check for issues "Contrast"
- Select one of the found contrast violations

## Demo
https://www.loom.com/share/614baef41ff1436c90ef353db13f804b

---

This PR was pushed through [GitStart](https://www.gitstart.com/), with contributions from @passy4ucj @WajihaNiazi, @hashimaziz46
